doc/rippled-example.cfg

```diff
@@ -987,23 +972,6 @@ pool.ntp.org
 [ips]
 r.ripple.com 51235
 
-# Public keys of the validators that this rippled instance trusts.  The latest
```

Comment:
I agree that the default shouldn't give up silently, however I think `[validator_file]` currently only works with an absolute path.
What if I change that so it can be either an absolute path or a filename that we search for in `CONFIG_DIR`? That way a default configuration of `[validators_file]` as `validators.txt` will work if validators.txt resides in the same directory as rippled.cfg

On the topic of giving up silently, does logging even work at the point where we check for `[validators_file]` because I'm not able to see them?
https://github.com/ripple/rippled/blob/develop/src/ripple/core/impl/Config.cpp#L459
Should those error logs be changed to throws?